doc/rbd/rbd-snapshot.rst

```diff
@@ -66,75 +73,94 @@ Create Snapshot
 ---------------
 
 To create a snapshot with ``rbd``, specify the ``snap create`` option, the pool
-name and the image name.  ::
+name, and the image name:
 
-	rbd snap create {pool-name}/{image-name}@{snap-name}
+.. prompt:: bash $
 
-For example:: 
+   rbd snap create {pool-name}/{image-name}@{snap-name}
 
-	rbd snap create rbd/foo@snapname
+For example:
+
+.. prompt:: bash $
+
+   rbd snap create rbd/foo@snapname
 	
 
 List Snapshots
 --------------
 
-To list snapshots of an image, specify the pool name and the image name. ::
+To list the snapshots of an image, specify the pool name and the image name:
 
-	rbd snap ls {pool-name}/{image-name}
+.. prompt:: bash $
 
-For example::
+   rbd snap ls {pool-name}/{image-name}
 
-	rbd snap ls rbd/foo
+For example:
 
+.. prompt:: bash $
 
-Rollback Snapshot
------------------
+   rbd snap ls rbd/foo
 
-To rollback to a snapshot with ``rbd``, specify the ``snap rollback`` option, the
-pool name, the image name and the snap name. ::
 
-	rbd snap rollback {pool-name}/{image-name}@{snap-name}
+Roll back Snapshot
+------------------
 
-For example::
+To roll back to a snapshot with ``rbd``, specify the ``snap rollback`` option,
+the pool name, the image name, and the snap name:
 
-	rbd snap rollback rbd/foo@snapname
+.. prompt:: bash $
 
+   rbd snap rollback {pool-name}/{image-name}@{snap-name}
 
-.. note:: Rolling back an image to a snapshot means overwriting 
-   the current version of the image with data from a snapshot. The 
-   time it takes to execute a rollback increases with the size of the 
-   image. It is **faster to clone** from a snapshot **than to rollback** 
-   an image to a snapshot, and is the preferred method of returning
-   to a pre-existing state.
+For example:
+
+.. prompt:: bash $
+
+   rbd snap rollback rbd/foo@snapname
+
+
+.. note:: Rolling back an image to a snapshot means overwriting the current
+   version of the image with data from a snapshot. The time it takes to execute
+   a rollback increases with the size of the image. It is **faster to clone**
+   from a snapshot **than to rollback** an image to a snapshot. Cloning from a
+   snapshot is the preferred method of returning to a pre-existing state.
 
 
 Delete a Snapshot
 -----------------
 
 To delete a snapshot with ``rbd``, specify the ``snap rm`` subcommand, the pool
```

Comment:
Ditto -- "To delete a snapshot, use `rbd snap rm` command and specify ...".

Reply:
Done.